### PR TITLE
vertical-navbar-modified

### LINKED ISSUE #886: Vertical navbar links
Can I make the navbar links verticals instead of horizontal?

Navbar
 
I think it will look nice If I made it vertical. Can you assign me this issue, please? @anushbhatia

### PR DESCRIPTION
**Pull request by MennaMagdy2019**
fixed #886 
I made the final changes on the navbar and I am very sorry for any mistake I made
Can you review it please? @smaranjitghose 
Vertical navbar

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Route, Switch, Redirect } from "react-router-dom";
 import { withRouter } from "react-router";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
-import NotFound from "./pages/404/notFound";
+import NotFound from "./pages/404/NotFound";
```
Comment (ashuvssut) [inline src/App.jsx:6]: DO NOT MERGE this PR yet

see my review at PR #900.
same applied to this one

Comment (ashuvssut) [inline src/App.jsx:6]: ✔ correct change.

I think we should merge this one before merging #900 and #899

Comment (MennaMagdy2019) [reply src/App.jsx:6]: > ✔ correct change.
> 
> I think we should merge this one before merging #900 and #899

I don't Know where is error 
Can you tell me please @ashuvssut?

Comment (ashuvssut) [reply src/App.jsx:6]: No error in this PR

Comment (ashuvssut) [reply src/App.jsx:6]: you PR #899 has the error

Comment (ashuvssut) [reply src/App.jsx:6]: image

Comment (ashuvssut) [reply src/App.jsx:6]: you are adding notFound.jsx

That must be NotFound.jsx

**Pull request closed without merging**